### PR TITLE
[Sprint Maintain] Feature Add Manager Continue

### PR DESCRIPTION
**Pull request by Tuanlvt**
Feature Add Manager Continue
Link redmine :https://dev.framgia.com/redmine/issues/128070

### CODE REVIEW

File: app/src/main/java/com/framgia/forder/screen/addmanagershop/ItemAddManagerInShopViewModel.java [hunk 1]
```diff
@@ -27,4 +34,28 @@ public String getName() {
...
+        notifyPropertyChanged(BR.visibleAddManager);
+    }
+
+    public void onClickAddManager() {
+        //Todo edit later
+    }
+
+    private void initValueCheckUser() {
+        if (mUser != null) {
+            if (mUser.getId() == mUserId) {
```
Comment (HyperionsThs) [inline app/src/main/java/com/framgia/forder/screen/addmanagershop/ItemAddManagerInShopViewModel.java]: change setVisibleAddManager(!mUser.getId()==mUserId);

File: app/src/main/java/com/framgia/forder/screen/addmanagershop/AddManagerShopPresenter.java [hunk 2]
```diff
@@ -76,4 +80,10 @@ public void onSafetyError(BaseException error) {
     public void addManagerInShop(int shopId, int userId) {
         //Todo Edit later
     }
+
+    @Override
+    public boolean checkUserId(int userId) {
+        User user = mUserRepository.getUser();
+        return user.getId() == userId;
+    }
```
Comment (nguyenhuy95dn) [inline app/src/main/java/com/framgia/forder/screen/addmanagershop/AddManagerShopPresenter.java]: remove if not used

File: app/src/main/java/com/framgia/forder/screen/addmanagershop/ItemAddManagerInShopViewModel.java (hunk 1 above)
Comment (nguyenhuy95dn) [inline app/src/main/java/com/framgia/forder/screen/addmanagershop/ItemAddManagerInShopViewModel.java]: should mUser.isCheckUser(mUserId)

File: app/src/main/res/layout/item_add_manager_in_shop.xml [hunk 3]
```diff
@@ -69,21 +70,52 @@
...
+                tools:ignore="RelativeOverlap"
+                >
+                <!--Todo edit later-->
+                <!--<ImageView-->
+                <!--android:layout_width="@dimen/dp_40"-->
+                <!--android:layout_height="@dimen/dp_40"-->
+                <!--android:src="@drawable/ic_manager_in_domain"-->
+                <!--android:visibility="@{viewModel.visibleOwner ? View.VISIBLE : View.GONE}"-->
+                <!--tools:ignore="ContentDescription,VectorDrawableCompat"-->
+                <!--/>-->
```
Comment (nguyenhuy95dn) [inline app/src/main/res/layout/item_add_manager_in_shop.xml]: remove

File: app/src/main/res/layout/item_add_manager_in_shop.xml [hunk 4]
```diff
@@ -69,21 +70,52 @@
...
+                <!--<Button-->
+                <!--android:layout_width="wrap_content"-->
+                <!--android:layout_height="@dimen/dp_40"-->
+                <!--android:background="@drawable/button_red"-->
+                <!--android:gravity="center"-->
+                <!--android:text="@string/delete"-->
+                <!--android:textAllCaps="false"-->
+                <!--android:textColor="@color/color_white"-->
+                <!--android:visibility="@{viewModel.visibleDeleteManager ? View.VISIBLE : View.GONE}"-->
+                <!--/>-->
```
Comment (nguyenhuy95dn) [inline app/src/main/res/layout/item_add_manager_in_shop.xml]: remove

**Pull request merged**